build/AzurePipelinesTemplates/ProjectReunion-CreateNugetPackage-Job.yml

```diff
@@ -54,7 +56,12 @@ jobs:
 
   - task: DownloadBuildArtifacts@0 
     inputs: 
-      artifactName: drop
+      artifactName: thin_nuget
```

Comment:
>thin_nuget [](start = 20, length = 10)

Nit: I assume "thin nuget" is a term we came up with internally (nothing comes up on google at least). We should probably find a good place in this repo to put a readme about what this refers too (no need to necessarily do it in this PR though)

Reply:
Correct, Eho coined those terms.  I expect before we ship we will want to come up with a nicer term than "thin" and "fat" anyways.  I was going to table that decision until we finalize on whether the AppX is actually going to ship inside the Thin Nuget or separately.